pkg/util/iptables/iptables.go

```diff
@@ -617,7 +621,7 @@ const (
 	opCreateChain operation = "-N"
 	opFlushChain  operation = "-F"
 	opDeleteChain operation = "-X"
-	opListChain   operation = "-L"
+	opListChain   operation = "-S"
```

Comment:
`-S` is better than `-nL` and I think is doing the same, without the dns reverse lookups

Reply:
So there's some trickiness here which is that iptables-nft 1.8.3 is not consistent with iptables-legacy about error codes in some edge cases. So you need to confirm that `iptables-nft -S NON-EXISTENT-CHAIN` and `iptables-legacy -S NON-EXISTENT-CHAIN` return the same thing... (Originally the monitoring code had used `-F` to check if the chain existed (since it's empty anyway, flushing would be a no-op if the chain existed, but would return an error if it didn't). But that didn't work because iptables-nft didn't actually return an error if the chain didn't exist..)

Reply:
seems is consistent, and is what is being used in go-iptables
```
root@kind-control-plane:/# iptables-legacy
iptables v1.8.3 (legacy): no command specified
Try `iptables -h' or 'iptables --help' for more information.
root@kind-control-plane:/# iptables-legacy -t nat -L NON
iptables: No chain/target/match by that name.
root@kind-control-plane:/# echo $?
1
root@kind-control-plane:/# iptables-nft
iptables v1.8.3 (nf_tables): no command specified
Try `iptables -h' or 'iptables --help' for more information.
root@kind-control-plane:/# iptables-nft -t nat -L NON
# Warning: iptables-legacy tables present, use iptables-legacy to see them
iptables: No chain/target/match by that name.
root@kind-control-plane:/# echo $?
1
```

Reply:
>     root@kind-control-plane:/# iptables-nft -t nat -L NON

`-L` is not `-S` :slightly_smiling_face:

Reply:
```
root@kind-control-plane:/#  iptables-legacy -t nat -S NON
iptables: No chain/target/match by that name.
root@kind-control-plane:/# echo $?
1
root@kind-control-plane:/# iptables-nft -t nat -S NON
# Warning: iptables-legacy tables present, use iptables-legacy to see them
iptables: No chain/target/match by that name.
root@kind-control-plane:/# echo $?
1
```
:facepalm: